python/dataproc_templates/kafka/README.md

```diff
@@ -0,0 +1,62 @@
+# Kafka To BigQuery
+
+Template for reading files from streaming Kafka topic and writing them to a BigQuery table.
+
+It uses the 
+  - [Spark BigQuery connector](https://cloud.google.com/dataproc-serverless/docs/guides/bigquery-connector-spark-example) for writing to BigQuery.
+
+  - [ Kafka 0.10+ Source For Structured Streaming](https://mvnrepository.com/artifact/org.apache.spark/spark-sql-kafka-0-10) for connecting list of broker connections.
+
+## Arguments
+
+* `kafka.to.bq.checkpoint.location`: Cloud Storage location for storing checkpoints during transfer (format: `gs://bucket/...`)
+* `kafka.to.bootstrap.servers`: List of kafka bootstrap servers (format: *'[x1.x2.x3.x4:port1,y1.y2.y3.y4:port2]')*
+* `kafka.to.bq.topic`: Topic names for respective kafka server
+* `kafka.to.bq.starting.offset`:  Offset to start reading from. Accepted values: "earliest", "latest" (streaming only), or json string """ {"topicA":{"0":23,"1":-1},"topicB":{"0":-2}} """
+* `kafka.to.bq.dataset`: Temporary bucket for the Spark BigQuery connector
+* `kafka.to.bq.table`: Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to append)
+* `kafka.to.bq.temp.bucket.name`: Name of bucket for temporary storage files (not location).
+* `kafka.to.bq.termination.timeout`: **(in seconds)** Waits for specified time in ms before termination of stream 
```

Comment:
Update the description, replace ms with seconds.

Reply:
Done